differ/differ.go

```diff
@@ -614,6 +614,16 @@ func closeNotifier() {
 func pushMetrics(metricsConf conf.MetricsConfiguration) {
 	err := PushMetrics(metricsConf)
 	if err != nil {
+		log.Err(err).Msg(metricsPushFailedMessage)
+		for i := metricsConf.Retries; i > 0; i-- {
+			time.Sleep(metricsConf.RetryAfter)
+			log.Info().Msgf("Push metrics. Retrying (%d/%d attempts left)", i, metricsConf.Retries)
+			err = PushMetrics(metricsConf)
```

Comment:
you seem to rewrite error from line #616

Reply:
It is indeed rewritten. But if we get to that point, it means there's still an error, and it might have changed.